recipes/gnuradio-iio/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "gnuradio-iio" %}
+{% set hash = "733c8a05e74b7d10fbaef502cc82d025ae35a1fb" %}
+{% set version = "0.3.20200508+gr38" %}
```

Comment:
Please build from a release tarball, not git

Reply:
I mentioned this in https://github.com/conda-forge/staged-recipes/pull/11688#issuecomment-634156747, but it's because there hasn't been a release yet that is compatible with `gnuradio=3.8`. If it's a blocker, I can drop this recipe for now until I can get upstream to make a release.

Reply:
Yeah, this is a blocker. Sorry. If you can drop it for now that would be easiest

Reply:
No problem, I can push harder to get them to make a release and until then I have my own package. Done now.